### DISCUSSION TITLE
¿How to change the lenguage?

### DISCUSSION THREAD

**Discussion by remixdance** [FlowiseAI/Flowise#479] [General]
Hi I´m usinge the lastest version of Flowise with a flow to answer question about some PDF´s  I´m using the 1.0.9 version of embeded chat but when I send my request on spanish the chat is alwys on English is there a way to change the lenguage. 
![image](https://github.com/FlowiseAI/Flowise/assets/42038961/d0b76fe9-c00b-4615-b578-1201bb15d2b2)

**Comment by marius-smg** [comment]
i had this problem as well that it occasionally gave answers in english although everything (including the documents) was in german. I believe this problem is on the side of ChatGPT and not Flowise. I had similar things happen when in used ChatGPT directly in the web interface. This problem existed for 2-3 weeks but in the last days my Flowise Application gave answers in the correct language. You can try to tinker around with the "Additional Paramters" and say that it only should answer in spanish. But other than that, i believe there is no direct problem with flowise, though i might be wrong.

**Reply by remixdance** [reply]
Ok thanks can you tell me what options did you use for "additional parameters" or in which modules did you use it to have the answer on german? In my case I need spanish but any information are useful 
thanks

**Reply by HenryHengZJ** [reply]
yeah its trial and error. https://github.com/FlowiseAI/Flowise/discussions/453

**Reply by jwalatirkey858-tech** [reply]
I documented a structured implementation blueprint for this. Let me know if you want it.

**Comment by johntowns752** [comment]
I finally gave https://mybabes.ai/en a proper spin and fuck me, it’s dangerously good. The videos hit different—stunning 4K-level clarity, perfect skin texture, realistic lighting, and movement so fluid you’d swear it was shot with a real camera. Zero of that plastic, glitchy AI feel. Just straight fire.
Customization is on another planet. You pick literally everything: face, age range, body shape, curves, cock size if that’s your thing, hair, tattoos, piercings, lingerie or naked, bedroom / outdoor / public vibe, lighting mood, camera angles, pacing, positions, intensity, moans, dialogue—you build the whole damn scene exactly how your brain pictured it. It’s addictive as hell.
The top-shelf renders and unlimited generations are paid, but let’s be real: the quality is so far above free crap that it feels like a steal. You get what you pay for and then some.
Completely anonymous, no real humans, no leaks, no awkward convos—just you unleashing whatever filthy fantasy you want in total privacy. Site’s fast, slick, stupidly easy to use, and they actually give a shit about data security.
If you’re done settling for boring, recycled porn and want stuff that’s 100% yours, custom-made, and looks this fucking good… this is it. One session and you’ll be clearing your schedule. Go on, treat yourself. You know you want to. 😈

**Reply by jwalatirkey858-tech** [reply]
I documented a structured implementation blueprint for this. Let me know if you want it.
